scripts/rv_expression_association/igll5_vep.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import hail as hl
+import numpy as np
+import pandas as pd
+from hail.methods import export_plink
+from cpg_utils.hail_batch import dataset_path, init_batch, output_path #, reference_path
+
+# object containing variants within a 50K window on either side of the IGLL5 gene
+MT = dataset_path('v0/IGLL5_50K_window.mt')
+VEP_HT = dataset_path('v0/IGLL5_50K_window_vep.ht')
+
+def main():
+    # read and densify object
+    init_batch()
+    mt = hl.read_matrix_table(MT)
+    mt = hl.experimental.densify(mt)
+
+    print(mt.count())
+
+    # filter out low quality variants
+    mt = mt.filter_rows(hl.len(hl.or_else(mt.filters, hl.empty_set(hl.tstr))) == 0)
+
+    # consider biallelic variants only (no multi-allelic, no ref-only)
+    mt = mt.filter_rows(hl.len(mt.alleles) == 2)
+    mt = mt.filter_rows(hl.is_snp(mt.alleles[0], mt.alleles[1]))
+
+    # annotate using VEP
+    vep_ht = hl.read_table(VEP_HT)
+    mt = mt.annotate_rows(vep=vep_ht[mt.row_key].vep)
+
+    print(mt.count())
+
+    # filter rare variants only (MAF < 5%)
+    mt = hl.variant_qc(mt)
+    rv_mt = mt.filter_rows(mt.variant_qc.AF[1] < 0.05)
+    rv_mt = rv_mt.filter_rows(rv_mt.variant_qc.AF[1] > 0)
+
+    print(rv_mt.count())
```

Comment:
We tend to prefer the `logging` library for logging stuff to the console, I'd probably encourage labels to go along with these print statements so they're not just numbers in stderr.

Reply:
Yes sorry, should be done now, but how do you deal with this when it's a table / multi-line? i.e. https://github.com/populationgenomics/tob-wgs/blob/get-variants/scripts/rv_expression_association/get_vep_variants.py#L58, which outputs
<img width="450" alt="Screen Shot 2022-09-27 at 10 05 12 am" src="https://user-images.githubusercontent.com/25035866/192447454-3e307501-6196-4cbb-9afc-6d7c3384d31d.png">